src/PlusCommon/vtkPlusConfig.cxx

```diff
@@ -417,6 +417,19 @@ PlusStatus vtkPlusConfig::LoadApplicationConfiguration()
     saveNeeded = true;
   }
 
+  // Read current tab
+  int currentTab = 0;

```

Comment:
Considering vtkPlusConfig is used for all Plus applications, and not those with a tabbed gui, perhaps this should be read/written a PlusServerLauncher::ReadConfiguration function? The data can still be saved in PlusConfig.xml

Reply:
Good point. I'll add this to PlusServerLauncher then.